mol_dqn/README.md

```diff
@@ -7,3 +7,65 @@ Molecules via Deep Reinforcement Learning; http://arxiv.org/abs/1810.08678.
 Note that the scripts are in an "under construction" state and will be updated
 shortly. The main library functions, such as the MDP definition in
 `chemgraph/mcts/molecules.py`, are of primary interest.
+
+Here are the commands to produce the experimental results:
+## Prepare
+
+### Install the `Contrib` module of `rdkit`
+
+```
+git clone https://github.com/rdkit/rdkit
+cp -R ./rdkit/Contrib/SA_Score ./chemgraph/dqn/py
+```
+
+## Single Property Optimization
+
+### Optimization of QED
+
+#### Naive DQN
+
+```
+python optimize_qed.py --model_dir="./save/optimize_qed/7/" --hparams="./configs/naive_dqn.json"
+```
+
+#### Bootstrap DQN
+##### Step 1
+```
+python optimize_qed.py --model_dir="./save/optimize_qed/7/" --hparams="./configs/bootstrap_dqn_step1.json"
+```
+##### Step 2
+```
+python optimize_qed.py --model_dir="./save/optimize_qed/7/" --hparams="./configs/bootstrap_dqn_step1.json"
+```
+### Optimization of logP
+
+#### Naive DQN
+
+```
+python optimize_logp.py --model_dir="./save/optimize_qed/7/" --hparams="./configs/naive_dqn.json"
+```
+
+#### Bootstrap DQN
+##### Step 1
+```
+python optimize_logp.py --model_dir="./save/optimize_qed/7/" --hparams="./configs/bootstrap_dqn_step1.json"
+```
+
+## Constraint Optimization
+
+### Naive DQN
+```
+python optimize_logp_of_800_molecules.py --model_dir="./save/optimize_800_bs/1/" --hparams="./configs/naive_dqn_opt_800.json" --similarity_constraint=0.0
+```
+### Bootstrap DQN
+```
+python optimize_logp_of_800_molecules.py --model_dir="./save/optimize_800_bs/1/" --hparams="./configs/bootstrap_dqn_opt_800.json" --similarity_constraint=0.0
+```
+
+## Multi-objective Optimization
+
+### Bootstrap DQN
+```
+python multi_obj_naive.py --model_dir="./save/multi_obj_naive/mol${1}/sim${2}/" --hparams="./configs/multi_obj_dqn.json" --start_molecule="CCN1c2ccccc2Cc3c(O)ncnc13" --target_molecule="CCN1c2ccccc2Cc3c(O)ncnc13" --similarity_weight=0.0
```

Comment:
This is unusual to have in the actual command line. Please create a more specific example that doesn't expect additional positional arguments.